src/main/java/org/sagebionetworks/bridge/udd/helper/SnsHelper.java

```diff
@@ -2,9 +2,9 @@
 
 import java.util.Map;
 
-import org.sagebionetworks.bridge.udd.accounts.AccountInfo;
-import org.sagebionetworks.bridge.udd.dynamodb.StudyInfo;
 import org.sagebionetworks.bridge.udd.s3.PresignedUrlInfo;
+import org.sagebionetworks.bridge.workerPlatform.bridge.AccountInfo;
+import org.sagebionetworks.bridge.workerPlatform.dynamodb.StudyInfo;
```

Comment:
I don't know about a package name that has a capital in it... I've never seen that before. In terms of the principle of least surprise, I'd expect it to be workerplatform.

Reply:
If we were creating a new package, I'd be inclined to change it to workerplatform (or maybe just worker). However, the package already exists and has a bunch of files in it, and I don't think we should rename it at this point.